modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -102,13 +102,13 @@ const transformToDate = (
 			return moment(date, [
 				getLocaleDateFormat(locale),
```

Comment:
@mlan0, please double check if we can get rid of this code.

Reply:
We can't. The moment() function needs to match the date string with at least one format that is provided in this array.
In the View Form, the initial date is in the format YYYY-MM-DD, and the getLocaleDateFormat(locale) is also necessary to the moment() because the predefined value input gives a date in the LocaleDateFormat.